<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manuals</class>
 <widget class="QWidget" name="Manuals">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>631</width>
       <height>831</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>631</width>
       <height>831</height>
      </size>
     </property>
     <widget class="QPushButton" name="intelDocsButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>470</y>
        <width>550</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="assemblyTutorialButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>560</y>
        <width>550</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadCheatsheetButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>650</y>
        <width>550</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="MainBackButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>740</y>
        <width>550</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>450</width>
        <height>450</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
